src/core/css/inc/vars.scss

```diff
@@ -16,6 +16,7 @@ $button-background-color: #0095dd;
 $base-color: #fff;
 $link-color: $blue-60;
 $warning: #d63920;
+$gradient-color: #f4f9ff;
```

Comment:
This should probably have a more specific name, like `showmorecard-gradient-color` or something.